### ISSUE TITLE
_N events scanned to T_ is now missing time component when T is recent

### ISSUE THREAD

**Issue by nblumhardt** [datalust/seq-tickets#1510]
Regression in 2021.4. The _N events scanned to T_ text shown while a search is in progress used to display the time component, e.g. "Tuesday ... at 4:31 pm", but since this was updated to respect user date and time formatting preferences, now shows the date only.

This is inconvenient when searching large volumes of data logged over a short time period. Ideally, the time component of T will be displayed if T is within the last 7 days.

**Comment by Suchiman** [comment]
I'd argue to always display the time component because there's more than enough space to display it and it makes it much easier to see how far query has been progressing (which is helpful if you want to abort the query, change something and then use the the `Now` range filter to resume where you left off).

**Comment by andreanime** [comment]
> I'd argue to always display the time component because there's more than enough space to display it and it makes it much easier to see how far the query has progressed (which is helpful if you want to abort the query, change something and then use the the `To` range filter to resume where you left off).

I agree. It's very helpful when you need to search a large amount of data, because when you don't even know how far was the query's progress, it becomes harder to make new filters, and try to reduce the time period you're searching.

**Comment by nblumhardt** [comment]
Fixed in the upcoming patch 👍
